src/status_im2/subs/chat/chats.cljs

```diff
@@ -152,6 +152,13 @@
  (fn [[chat-id inputs]]
    (get-in inputs [chat-id :metadata :sending-image])))
 
+(re-frame/reg-sub
```

Comment:
we already have :chats/current-chat-inputs, bettter to also have current-chat-input-metada based on it, so you can use it for selected photos

Reply:
it could be used in :chats/sending-image as well